doc/proposals/ansible-operator.md

```diff
@@ -0,0 +1,78 @@
+## Ansible Operator Proposal
+
+### Background
+
+Not everyone is a golang developer, and therefore gaining adoption for the operator-sdk is capped by the number of golang developers. Also, tooling for kubernetes in other languages is lacking support for things such as informers,  caches, and listers.  
+
+Operators purpose is to codify the operations of an application on kubernetes. [Ansible](https://www.ansible.com/) is already an industry standard tool for automation and is a good fit for the kind of work that kubernetes operators need to do. Adding the ability for users of the SDK to choose which between ansible and golang to follow will increase the number of potential users, and will grant existing users even more behavior. 
+
+### Goals
+
+The goal of the Ansible Operator will be to create a fully functional framework for Ansible developers to create operators. It will also expose a library for golang users to use ansible in their operator if they so choose. These two goals in conjunction will allow users to select the best technology for their project or skillset. 
+
+### New Operator Type
+
+This proposal creates a new type of operator called `ansible`.  The new type is used to tell the tooling to act on that type of operator. 
+
+### Package Structure
+Packages will be added to the operator-sdk. These packages are designed to be usable by the end user if they choose to and should have a well documented public API. The proposed packages are:
+* /operator-sdk/pkg/ansible/controller
+  * Will contain the ansible operator controller.
+  * Will contain a exposed reconciler. But the default `Add` method will use this reconciler.
```

Comment:
> But the default `Add` method will use this reconciler

@shawn-hurley Just so I understand this correctly, this controller is expected to be added to the manager in the hybrid case right?
likely in cmd/manager/main.go:
```Go
import ac "github.com/operator-framework/operator-sdk/pkg/ansible/controller"
...
// Add the manage
ac.Add(mgr, ac.Options{...})
```
So users can either call `Add()` to add the default ansible controller to their manager. 
Or create their own controller and use the default reconciler (`AnsibleOperatorReconciler` from #461 )

Reply:
*an exposed

Reply:
@hasbro17 that is what we are thinking as well +1